deployment/computer-vision/base64/config.sample.ini

```diff
@@ -0,0 +1,9 @@
+; Rename this file to config.ini.
+; Replace the information below with the correct values.
+[APP]
+VERTA_HOST = host.verta.ai
+VERTA_EMAIL = user@.verta.ai
```

Comment:
minor: no "." after @

Reply:
This file was removed, since the Colab notebook should work without mounting.